pkg/controller/scyllacluster/resource_test.go

```diff
@@ -579,29 +579,24 @@ func TestStatefulSetForRack(t *testing.T) {
 		}
 	}
 
-	newBasicStatefulSetLabels := func() map[string]string {
+	newBasicStatefulSetLabels := func(ordinal int) map[string]string {
 		return map[string]string{
 			"app":                          "scylla",
 			"app.kubernetes.io/managed-by": "scylla-operator",
 			"app.kubernetes.io/name":       "scylla",
 			"scylla/cluster":               "basic",
 			"scylla/datacenter":            "dc",
 			"scylla/rack":                  "rack",
-			"scylla/rack-ordinal":          "0",
+			"scylla/scylla-version":        "",
+			"scylla/rack-ordinal":          fmt.Sprintf("%d", ordinal),
```

Comment:
nit: strconv.Itoa

Reply:
why?

Reply:
avoids parsing the format string (but we can spare the extra cycle in test :) )